### ISSUE TITLE
Compatible with RHEL9 ?

### ISSUE THREAD

**Issue by eylenth** [jhoblitt/puppet-ganglia#124]
Hi 

It seems that the puppet module is not supported for RHEL9. https://github.com/jhoblitt/puppet-ganglia/blob/master/manifests/params.pp#L60

Will there be any plans to support the puppet module for RHEL 9?

Thomas

**Comment by jhoblitt** [comment]
There were no rpms in EPEL9 when the module was last updated. Has that changed?

**Comment by eylenth** [comment]
Yes, now there are rpms available in EPEL9:
```sh
sudo dnf search ganglia
Updating Subscription Management repositories.
Extra Packages for Enterprise Linux 9 - x86_64                                                                                                                                   9.0 MB/s |  18 MB     00:01
Last metadata expiration check: 0:00:05 ago on Tue 11 Jul 2023 03:46:19 PM CEST.
========================================================================================= Name Exactly Matched: ganglia =========================================================================================
ganglia.x86_64 : Distributed Monitoring System
======================================================================================== Name & Summary Matched: ganglia ========================================================================================
ganglia-devel.x86_64 : Ganglia Library
ganglia-gmetad.x86_64 : Ganglia Metadata collection daemon
ganglia-gmond.x86_64 : Ganglia Monitoring daemon
ganglia-web.x86_64 : Ganglia Web Frontend
```

**Comment by jhoblitt** [comment]
Great. Please feel free to open a PR.

**Comment by eylenth** [comment]
PR is created #125

**Comment by jhoblitt** [comment]
Resolved in #125
